tests/unit/amo/components/Errors/TestNotAuthorized.js

```diff
@@ -31,6 +31,7 @@ describe(__filename, () => {
   it('renders a not authorized error', () => {
     const root = render();
 
-    expect(root.find(Card)).toHaveProp('header', 'Not Authorized');
+    expect(root.find(ErrorComponent)).toHaveProp('code', 401);
+    expect(root).toHaveProp('header', 'Not Authorized');
```

Comment:
This should be `expect(root.find(ErrorComponent)).toHaveProp('header', 'Not Authorized');`